src/test/unit/EigenPod-PodManagerUnit.t.sol

```diff
@@ -1,53 +1,666 @@
+            eigenPodManager,
+            MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR,
+            GOERLI_GENESIS_TIME
+        );
+
+        eigenPodBeacon = new UpgradeableBeacon(address(podImplementation));
+
+        // Deploy EigenPodManager implementation
+        eigenPodManagerWrapper = new EigenPodManagerWrapper(
+            ethPOSMock,
+            eigenPodBeacon,
+            strategyManagerMock,
+            slasherMock,
+            delegationManagerMock
+        );
+
+        eigenLayerProxyAdmin.upgradeAndCall(
+            TransparentUpgradeableProxy(payable(address(eigenPodManager))),
+            address(eigenPodManagerWrapper),
+            abi.encodeWithSelector(
+                EigenPodManager.initialize.selector,
+                type(uint256).max /*maxPods*/,
+                beaconChainOracle,
+                initialOwner,
+                pauserRegistry,
+                0 /*initialPausedStatus*/
+            )
+        );
+
+        // Below is a hack to get the eigenPod address that proofs prove against
+        
+        // Deploy Proxy same way as EigenPodManager does
+        eigenPod = EigenPod(payable(
+            Create2.deploy(
+                0,
+                bytes32(uint256(uint160(address(podOwner)))),
+                // set the beacon address to the eigenPodBeacon
+                abi.encodePacked(beaconProxyBytecode, abi.encode(eigenPodBeacon, ""))
+        )));
+
+        // Etch the eigenPod code to the address for which proofs are generated
+        bytes memory code = address(eigenPod).code;
+        cheats.etch(podAddress, code);
+        eigenPod = EigenPod(payable(podAddress));
+
+        // Store the eigenPodBeacon address in the eigenPod beacon proxy
+        bytes32 beaconSlot = 0xa3f0ad74e5423aebfd80d3ef4346578335a9a72aeaee59ff6cb3582b35133d50;   
+        cheats.store(address(eigenPod), beaconSlot, bytes32(uint256(uint160(address(eigenPodBeacon)))));
+
+        // Initialize pod
+        eigenPod.initialize(address(podOwner));
+
+        // Set storage in EPM
+        EigenPodManagerWrapper(address(eigenPodManager)).setPodAddress(podOwner, eigenPod);
+    }
+}
+
+contract EigenPod_PodManager_UnitTests_EigenPodPausing is EigenPod_PodManager_UnitTests {
+    /**
+     * 1. verifyBalanceUpdates revert when PAUSED_EIGENPODS_VERIFY_BALANCE_UPDATE set
+     * 2. verifyAndProcessWithdrawals revert when PAUSED_EIGENPODS_VERIFY_WITHDRAWAL set
+     * 3. verifyWithdrawalCredentials revert when PAUSED_EIGENPODS_VERIFY_CREDENTIALS set
+     * 4. activateRestaking revert when PAUSED_EIGENPODS_VERIFY_CREDENTIALS set
+     */
+
+    /// @notice Index for flag that pauses creation of new EigenPods when set. See EigenPodManager code for details.
+    uint8 internal constant PAUSED_NEW_EIGENPODS = 0;
+    /**
+     * @notice Index for flag that pauses all withdrawal-of-restaked ETH related functionality `
+     * function *of the EigenPodManager* when set. See EigenPodManager code for details.
+     */
```

Comment:
Nit: can we standardize these comment notations, can just use `///`  in one line